### ISSUE TITLE
onpause doesn't fire in chrome

### ISSUE THREAD

**Issue by techsin** [WICG/speech-api#82]
```
speechSynthesis.cancel();
text = 'This is a story about a man who once lived in a mountains far into Norther region of our country in a village which was known for its tasty food. He always had a dream as a kid to travel around the world.';
utterance = new SpeechSynthesisUtterance(text);
utterance.voice = speechSynthesis.getVoices().filter(x => x.lang == 'en-US' && x.voiceURI == 'Google US English')[0]
utterance.onpause = e => console.log(e);
speechSynthesis.speak(utterance);

setTimeout(function(){
  speechSynthesis.pause(utterance);
}, 3000);

```

but if you set to this voice it works

```utterance.voice = speechSynthesis.getVoices().filter(x => x.lang == 'en-US' && x.name == 'Alex')[0]```

Mac
Chrome Version 86.0.4240.75 (Official Build) (x86_64)

**Comment by foolip** [comment]
Thanks for the report @techsin!

I've tested this on Chrome 88 on macOS and can confirm the behavior. I believe it has something to do with the fact that the 'Google US English' voice is `localService: false` while the 'Alex' voice is `localService: true`. In other words, something probably isn't wired up correctly for remote speech synthesis.

I've filed https://bugs.chromium.org/p/chromium/issues/detail?id=1178713 with a modified test case. Since this is the spec issue tracker and the problem seems to be with Google Chrome, I'll go ahead and close this issue.
